clinica/iotools/converters/adni_to_bids/adni_modalities/adni_t1.py

```diff
@@ -180,10 +181,10 @@ def compute_t1_paths(source_dir, csv_dir, dest_dir, subjs_list, conversion_dir):
         # ('022_S_5004', 'bl'),
         # ('022_S_5004', 'm03'),
         # Several images: T1wa ...
-        ("006_S_4485", "m84"),
-        ("029_S_2395", "m72"),
-        ("114_S_6039", "bl"),
-        ("016_S_4952", "m48"),
+        #("006_S_4485", "m84"),
```

Comment:
I think these changes need to be in a separate PR as they do not address a problem related to the the changes in the nibabel function